tests/test_quantity.py

```diff
@@ -0,0 +1,590 @@
+    assert conversion_output.value == pytest.approx(101325.0)
+
+
+def test_pdl():
+    x = q.Quantity(1, "pdl")
+    conversion_output = x.to("N")
+    assert conversion_output.value == pytest.approx(0.13826, 0.0001)
+
+
+def test_ozf():
+    x = q.Quantity(1, "ozf")  # force_ounce
+    conversion_output = x.to("N")
+    assert conversion_output.value == pytest.approx(0.27802, 0.0001)
+
+
+def test_lbf():
+    x = q.Quantity(1, "lbf")  # force_pound
+    conversion_output = x.to("N")
+    assert conversion_output.value == pytest.approx(4.44820, 0.00002)
+
+
+def test_kgf():
+    x = q.Quantity(1, "kgf")
+    conversion_output = x.to("N")
+    assert conversion_output.value == pytest.approx(9.806805, 0.0002)
+
+
+def test_tonf():
+    x = q.Quantity(1, "tonf")
+    conversion_output = x.to("N")
+    assert conversion_output.value == pytest.approx(9964.12914, 0.1)
+
+
+def test_cal():
+    x = q.Quantity(1, "cal")
+    conversion_output = x.to("J")
+    assert conversion_output.value == pytest.approx(4.1868, 0.002)
+
+
+def test_kcal():
+    x = q.Quantity(1, "kcal")
+    conversion_output = x.to("J")
+    assert conversion_output.value == pytest.approx(4186.8, 2)
+
+
+def test_hp():
+    x = q.Quantity(1, "hp")
+    conversion_output = x.to("W")
+    assert conversion_output.value == pytest.approx(745.70, 0.01)
+
+
+def test_ohm():
+    x = q.Quantity(1, "ohm cm")
+    conversion_output = x.to("ohm m")
+    assert conversion_output.value == 0.01
+
+
+def test_hp_h():
+    x = q.Quantity(1, "hp h")  # hp_hour
+    conversion_output = x.to("J")
+    assert conversion_output.value == pytest.approx(2.6845 * 10**6, 19)
+
+
+def test_erg():
+    x = q.Quantity(1, "erg")
+    conversion_output = x.to("J")
+    assert conversion_output.value == pytest.approx(1.0 * 10**-7)
+
+
+def test_energy_density():
+    x = q.Quantity(1, "BTU ft^-2")
+    conversion_output = x.to("J m^-2")
+    assert conversion_output.value == pytest.approx(11356.36, 0.2)
+
+
+def test_degree_rankine():
+    x = q.Quantity(1, "BTU lb^-1 degR^-1")
+    conversion_output = x.to("J kg^-1 degK^-1")
+    assert conversion_output.value == pytest.approx(4186.69, 0.2)
+
+
+def test_degree_fahrenheit():
+    x = q.Quantity(1, "BTU lb^-1 degF^-1")
+    conversion_output = x.to("J kg^-1 degK^-1")
+    assert conversion_output.value == pytest.approx(4186.69, 0.2)
+
+
+def test_degree_celsius():
+    x = q.Quantity(1, "cal g^-1 degC^-1")
+    conversion_output = x.to("J kg^-1 degK^-1")
+    assert conversion_output.value == pytest.approx(4186.69, 2)
+
+
+def test_mol():
+    x = q.Quantity(1, "lb mol ft^-3 s^-1")
+    conversion_output = x.to("kg mol m^-3 s^-1")
+    assert conversion_output.value == pytest.approx(16.01846, 0.000003)
+
+
+def test_rpm():
+    x = q.Quantity(1, "rpm")
+    conversion_output = x.to("rad s^-1")
+    assert conversion_output.value == pytest.approx(0.1047198, 0.000001)
+
+
+def test_particles_per_gm():
+    x = q.Quantity(1, "1^15particles g^-1")
```

Comment:
@hpohekar what happens when you multiply two quantities of this type together?

Reply:
We are getting following result after multiplying 2 quantities of this type together.

![image](https://user-images.githubusercontent.com/106588300/186113869-c5fc4e84-4582-49a1-af17-943df49a9fa6.png)

Reply:
per gram per kilogram doesn't give me a good feeling. Hopefully this is a side effect of supporting the special units above, which we are now removing. Can you check the something like that doesn't appear when e.g. multiply two "g^-1" quantities?

Reply:
No this is not appearing while dealing with standard units.

![image](https://user-images.githubusercontent.com/106588300/186126936-e7b837e7-85f6-4db3-972b-1d61efcb4166.png)